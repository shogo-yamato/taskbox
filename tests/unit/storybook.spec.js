// テスト用にStoryのスナップショットを作成
// あくまでスナップショット作成時とDOMの見た目がずれていないかを確認するだけ

import initStoryshots from "@storybook/addon-storyshots";

initStoryshots();
