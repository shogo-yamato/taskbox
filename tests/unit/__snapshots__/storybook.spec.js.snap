// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots PureInboxScreen Default 1`] = `
<div>
  <div
    class="page lists-show"
  >
    <nav>
      <h1
        class="title-page"
      >
        <span
          class="title-wrapper"
        >
          タスク一覧
        </span>
      </h1>
    </nav>
     
    <div
      class="list-items"
    >
      <!---->
       
      <div
        class="list-item TASK_INBOX"
      >
        <label
          class="checkbox"
        >
          <input
            disabled="disabled"
            name="checked"
            type="checkbox"
          />
           
          <span
            class="checkbox-custom"
          />
        </label>
         
        <div
          class="title"
        >
          <input
            placeholder="タイトルを入力してください"
            readonly="readonly"
            type="text"
          />
        </div>
         
        <div
          class="actions"
        >
          <a>
            <span
              class="icon-star"
            />
          </a>
        </div>
      </div>
      <div
        class="list-item TASK_INBOX"
      >
        <label
          class="checkbox"
        >
          <input
            disabled="disabled"
            name="checked"
            type="checkbox"
          />
           
          <span
            class="checkbox-custom"
          />
        </label>
         
        <div
          class="title"
        >
          <input
            placeholder="タイトルを入力してください"
            readonly="readonly"
            type="text"
          />
        </div>
         
        <div
          class="actions"
        >
          <a>
            <span
              class="icon-star"
            />
          </a>
        </div>
      </div>
      <div
        class="list-item TASK_INBOX"
      >
        <label
          class="checkbox"
        >
          <input
            disabled="disabled"
            name="checked"
            type="checkbox"
          />
           
          <span
            class="checkbox-custom"
          />
        </label>
         
        <div
          class="title"
        >
          <input
            placeholder="タイトルを入力してください"
            readonly="readonly"
            type="text"
          />
        </div>
         
        <div
          class="actions"
        >
          <a>
            <span
              class="icon-star"
            />
          </a>
        </div>
      </div>
      <div
        class="list-item TASK_INBOX"
      >
        <label
          class="checkbox"
        >
          <input
            disabled="disabled"
            name="checked"
            type="checkbox"
          />
           
          <span
            class="checkbox-custom"
          />
        </label>
         
        <div
          class="title"
        >
          <input
            placeholder="タイトルを入力してください"
            readonly="readonly"
            type="text"
          />
        </div>
         
        <div
          class="actions"
        >
          <a>
            <span
              class="icon-star"
            />
          </a>
        </div>
      </div>
      <div
        class="list-item TASK_INBOX"
      >
        <label
          class="checkbox"
        >
          <input
            disabled="disabled"
            name="checked"
            type="checkbox"
          />
           
          <span
            class="checkbox-custom"
          />
        </label>
         
        <div
          class="title"
        >
          <input
            placeholder="タイトルを入力してください"
            readonly="readonly"
            type="text"
          />
        </div>
         
        <div
          class="actions"
        >
          <a>
            <span
              class="icon-star"
            />
          </a>
        </div>
      </div>
      <div
        class="list-item TASK_INBOX"
      >
        <label
          class="checkbox"
        >
          <input
            disabled="disabled"
            name="checked"
            type="checkbox"
          />
           
          <span
            class="checkbox-custom"
          />
        </label>
         
        <div
          class="title"
        >
          <input
            placeholder="タイトルを入力してください"
            readonly="readonly"
            type="text"
          />
        </div>
         
        <div
          class="actions"
        >
          <a>
            <span
              class="icon-star"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots PureInboxScreen Error 1`] = `
<div>
  <div
    class="page lists-show"
  >
    <div
      class="wrapper-message"
    >
      <span
        class="icon-face-sad"
      />
       
      <div
        class="title-message"
      >
        無念...
      </div>
       
      <div
        class="subtitle-message"
      >
        エラーが発生しました。
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots PureTaskList Default 1`] = `
<div
  style="padding: 3rem;"
>
  <div
    class="list-items"
  >
    <!---->
     
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots PureTaskList Empty 1`] = `
<div
  style="padding: 3rem;"
>
  <div
    class="list-items"
  >
    <div
      class="wrapper-message"
    >
      <span
        class="icon-check"
      />
       
      <div
        class="title-message"
      >
        タスクはありません。
      </div>
       
      <div
        class="subtitle-message"
      >
        やったね！
      </div>
    </div>
     
  </div>
</div>
`;

exports[`Storyshots PureTaskList Loading 1`] = `
<div
  style="padding: 3rem;"
>
  <div
    class="list-items"
  >
    <div
      class="loading-item"
    >
      <span
        class="glow-checkbox"
      />
       
      <span
        class="glow-text"
      >
        <span>
          タダイマ
        </span>
         
        <span>
          ヨミコミチュウ
        </span>
         
        <span>
          デス
        </span>
      </span>
    </div>
    <div
      class="loading-item"
    >
      <span
        class="glow-checkbox"
      />
       
      <span
        class="glow-text"
      >
        <span>
          タダイマ
        </span>
         
        <span>
          ヨミコミチュウ
        </span>
         
        <span>
          デス
        </span>
      </span>
    </div>
    <div
      class="loading-item"
    >
      <span
        class="glow-checkbox"
      />
       
      <span
        class="glow-text"
      >
        <span>
          タダイマ
        </span>
         
        <span>
          ヨミコミチュウ
        </span>
         
        <span>
          デス
        </span>
      </span>
    </div>
    <div
      class="loading-item"
    >
      <span
        class="glow-checkbox"
      />
       
      <span
        class="glow-text"
      >
        <span>
          タダイマ
        </span>
         
        <span>
          ヨミコミチュウ
        </span>
         
        <span>
          デス
        </span>
      </span>
    </div>
    <div
      class="loading-item"
    >
      <span
        class="glow-checkbox"
      />
       
      <span
        class="glow-text"
      >
        <span>
          タダイマ
        </span>
         
        <span>
          ヨミコミチュウ
        </span>
         
        <span>
          デス
        </span>
      </span>
    </div>
    <div
      class="loading-item"
    >
      <span
        class="glow-checkbox"
      />
       
      <span
        class="glow-text"
      >
        <span>
          タダイマ
        </span>
         
        <span>
          ヨミコミチュウ
        </span>
         
        <span>
          デス
        </span>
      </span>
    </div>
     
  </div>
</div>
`;

exports[`Storyshots PureTaskList With Pinned Tasks 1`] = `
<div
  style="padding: 3rem;"
>
  <div
    class="list-items"
  >
    <!---->
     
    <div
      class="list-item TASK_PINNED"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="disabled"
          name="checked"
          type="checkbox"
        />
         
        <span
          class="checkbox-custom"
        />
      </label>
       
      <div
        class="title"
      >
        <input
          placeholder="タイトルを入力してください"
          readonly="readonly"
          type="text"
        />
      </div>
       
      <div
        class="actions"
      >
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Task Archived 1`] = `
<div
  class="list-item TASK_ARCHIVED"
>
  <label
    class="checkbox"
  >
    <input
      disabled="disabled"
      name="checked"
      type="checkbox"
    />
     
    <span
      class="checkbox-custom"
    />
  </label>
   
  <div
    class="title"
  >
    <input
      placeholder="タイトルを入力してください"
      readonly="readonly"
      type="text"
    />
  </div>
   
  <div
    class="actions"
  >
    <!---->
  </div>
</div>
`;

exports[`Storyshots Task Default 1`] = `
<div
  class="list-item TASK_INBOX"
>
  <label
    class="checkbox"
  >
    <input
      disabled="disabled"
      name="checked"
      type="checkbox"
    />
     
    <span
      class="checkbox-custom"
    />
  </label>
   
  <div
    class="title"
  >
    <input
      placeholder="タイトルを入力してください"
      readonly="readonly"
      type="text"
    />
  </div>
   
  <div
    class="actions"
  >
    <a>
      <span
        class="icon-star"
      />
    </a>
  </div>
</div>
`;

exports[`Storyshots Task Pinned 1`] = `
<div
  class="list-item TASK_PINNED"
>
  <label
    class="checkbox"
  >
    <input
      disabled="disabled"
      name="checked"
      type="checkbox"
    />
     
    <span
      class="checkbox-custom"
    />
  </label>
   
  <div
    class="title"
  >
    <input
      placeholder="タイトルを入力してください"
      readonly="readonly"
      type="text"
    />
  </div>
   
  <div
    class="actions"
  >
    <a>
      <span
        class="icon-star"
      />
    </a>
  </div>
</div>
`;
